tests/test_measureCrosstalk.py

```diff
@@ -0,0 +1,128 @@
+#
+# LSST Data Management System
+# Copyright 2008-2017 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+#
+
+import unittest
+import numpy as np
+
+import lsst.utils.tests
+from lsst.cp.pipe.measureCrosstalk import (MeasureCrosstalkTask, MeasureCrosstalkConfig)
+import lsst.ip.isr.isrMock as isrMock
+
+
+class MeasureCrosstalkTaskCases(lsst.utils.tests.TestCase):
+
+    def setup_measureCrosstalk(self, isTrimmed=False, nSources=8):
+        """Generate a simulated set of exposures and test the measured
+        crosstalk matrix.
+
+        Parameters
+        ----------
+        isTrimmed : `bool`, optional
+            Should the simulation use trimmed or untrimmed raw
+            exposures?
+        nSources : `int`, optional
+            Number of random simulated sources to generate in the
+            simulated exposures.
+
+        Returns
+        -------
+        coeffErr : `np.ndarray`
+            Array of booleans indicating if the measured and expected
+            crosstalk ratios are smaller than the measured uncertainty
+            in the crosstalk ratio.
+        """
+        mockTask = isrMock.CalibratedRawMock()
+        mockTask.config.rngSeed = 12345
+        mockTask.config.doGenerateImage = True
+        mockTask.config.doAddSky = True
+        mockTask.config.doAddSource = True
+        mockTask.config.doAddCrosstalk = True
```

Comment:
It's defined in `ip_isr`/`isrMock.py` as an 8x8 matrix that's largely zero.  It's not currently easily configurable, but it would be good to make it so.  I've filed DM-26029 for that work.

Reply:
Left a comment there instead of here, but that sounds great.